### ISSUE TITLE
Make ADD port index mandatory

### ISSUE THREAD

**Issue by ronag** [CasparCG/server#816]
Don't automatically make up indices. This can break stuff. But it's the kind of logic that could be moved out from the server to client side.

@5opr4ni what do you think?

i.e.

```
ADD 1-1 SCREEN
```

instead of

```
ADD 1 SCREEN
```

This is problematic in particular for the ffmeg_consumer that magically makes up indices.

**Comment by jesperstarkar** [comment]
How will this work? Will `ADD 1-1 SCREEN` only output layer 1 channel 1?

**Comment by ronag** [comment]
the second value is just an identifier. @5opr4ni was telling me that this could be confusing... but it is actually how it works today the `1` version is just a shorthand.

**Comment by jesperstarkar** [comment]
On the producer side, yes, `1`  means `1-0`, but do we have any consumers at all that takes a single layer? The layer re-route does, but not the others? Or am I missing a cool feature here?

**Comment by ronag** [comment]
nono, it has nothing to do with layers at all... it's just an port index so you can remove the consumer, e.g.
````
ADD 1-888 myskdjfnsdkf.mxf
REMOVE 1-888
```

**Comment by ronag** [comment]
The layer routing to consumers is kind of cool but can be achieved with channels + routing.

**Comment by jesperstarkar** [comment]
Ah, consumer index. Got it! Sorry about confusing, thanks for the explanation.

**Comment by jesperstarkar** [comment]
How about a new syntax to distinguish this from layers if it gets mandatory? `1:1`, `1:888`?

**Comment by TomKaltz** [comment]
I don't think it needs to be mandatory but we do need some way of getting information about running consumers.  We cannot do AMCP INFO nor get consumer information via OSC in 2.2 currently.  One of those needs to be added back in.
